trin-history/src/storage.rs

```diff
@@ -107,7 +104,10 @@ pub mod test {
                 PortalStorageConfig::new(CAPACITY_MB, temp_dir.path().to_path_buf(), node_id)
                     .unwrap();
             let mut storage = HistoryStorage::new(storage_config).unwrap();
-            let content_key = IdentityContentKey::random();
+            let prefixes = [0u8, 1u8, 2u8, 3u8];
+            let mut raw_content_key: Vec<u8> = IdentityContentKey::random().to_bytes();
+            raw_content_key.insert(0, *prefixes.choose(&mut rand::thread_rng()).unwrap());
+            let content_key = HistoryContentKey::try_from(raw_content_key).unwrap();
```

Comment:
nit: It is weird use bytes from random `IdentityContentKey` to create random `HistoryContentKey`.

I would say just just pick random prefix (as you do now) and then random 32 bytes. Maybe even implement this directly in `ethportal-api/src/types/content_key/history.rs`.